locale/ar_IQ/countries.xml

```diff
@@ -0,0 +1,263 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE countries SYSTEM "../../dtd/countries.dtd">
+
+<!--
+  * locale/ar_IQ/countries.xml
+  *
+  * Copyright (c) 2013-2016 Simon Fraser University Library
+  * Copyright (c) 2000-2016 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Localized list of countries. Based on ISO 3166-1.
+  -->
+<countries>
```

Comment:
On the other hand: We have a script creating the countries.xml file automatically, using the country names from the Debian project. So this probably does not need updating, because it will be replaced in the near future. I just discovered that there was no warning in the English file to not translate the file, sorry for that.